Detectors/ITSMFT/common/reconstruction/src/DigitPixelReader.cxx

```diff
@@ -224,6 +232,34 @@ bool DigitPixelReader::getNextChipData(ChipPixelData& chipData)
   return true;
 }
 
+//______________________________________________________________________________
+void DigitPixelReader::squashNeighbours(const uint16_t& iROF, const int& iDigit, const int& maxDigits, ChipPixelData& chipData)
```

Comment:
I discovered that there were some cases where the squasher was not working as I intended to do.
To simplify, it was just merging pixels in the next readout frame that would've been masked (if masking was activated). But in the case of clusters larger than the frame defined by `mMaxSquashDist`, it was just chopping them and redistributing them across the two readout frames.

I will open a JIRA ticked to append all the screenshots and discuss them there in case.

Reply:
Good, is there a cpu penalty for the fix?

Reply:
On a TF of PbPb 29kHz (543918) I get a 2% more.
Former squasher, fixed: `Cpu: 4.91 Real: 4.90444 s in 2 slots`
Recursive squasher, new: `Cpu: 5.01 Real: 5.00655 s in 2 slots`, 3 per-mille less clusters.

Reply:
Thanks! This is not critical :)